code/datums/abilities/flock/flockmind.dm

```diff
@@ -184,42 +264,37 @@
 	for(var/obj/machinery/door/airlock/A in range(10, holder.owner))
```

Comment:
This is a note for me mostly and you shouldn't worry about it right now

need to test/ask around if `range` is optimized for typechecks; if it's not this should iterate over `area.machines` and range check